populateStds.js

```diff
@@ -176,9 +179,30 @@ ${headerLegend}
         slines.push('| Version  | Type  | State   | stabilized | deprecated |')
         slines.push('| -------- | ----- | ------- | ---------- | ---------- |')
         var link = `[scs-${adrId}](/standards/${track.toLowerCase()}/scs-${adrId})`
-        var versionList = ['draft', 'stable', 'effective', 'deprecated'].map(
-            (column) => mkLinkList(versions.filter((v) => v.state[column])) || '-'
-        ).join(' | ')
+        var versionList = ['draft', 'effective', 'deprecated']
+        .map(
+            (column) =>
+              mkLinkList(
+                versions
+                  .filter((v) => v.state[column])
+                  .map((v) => {
+                    // for the effective column, check if there is a "future" equivalent
+                    // if so, this state will apply in the future and is marked with
+                    // an * for "futureEffective" and with a † for "futureDeprecated"
+                    return {
+                      ...v,
+                      version:
+                        v.version +
+                        (v.state.futureEffective
+                          ? '*'
+                          : v.state.futureDeprecated
+                          ? '&#8224;'
+                          : '')
+                    }
+                  })
+              ) || '-'
```

Comment:
I would prefer to have this logic in mkLinkList itself. This would have multiple advantages:

- this computation here can remain a three-liner
- the version field would not have to be changed (it has a very specific meaning)
- and the asterisk/dagger could be rendered _outside_ the link (as I think it should be)

Reply:
Oh, and one more advantage: the asterisks/daggers would then also be added for the supplements without changing line 221 (now; formerly 197).

Reply:
I agree, changed it.